release.json

```diff
@@ -5,7 +5,7 @@
         "6": "6.53.0",
         "7": "7.59.0"
     },
-    "nightly": {
+    "a6": {
```

Comment:
Is there a point behind keeping this ? I thought we were not releasing agent 6 from those branches anymore